### ISSUE TITLE
Accessibility: MAS40B: Inspect: Localized control type property is wrong for "Search" button

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#48]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open https://marketplace.visualstudio.com/VSCode in EDGE
2. Navigate to the Search button available under the "Extensions for the Visual Studio family of products" header 
3. Observe the "Localized Control Type" property in Inspect

**Actual Result:**
The "Localized Control Type" property is "Group"

**Narrator Behaviour:**
Narrator announces as "Search Group"

**Expected Result:**
The "Localized Control Type" property should be button
Eg: "Search Visual Studio Code Extensions" Button

**User Impact:**
Visually Impaired users will not be able to understand and will be confused about what control type the search is.

**Screenshots:**
![localized control type of search button is group](https://cloud.githubusercontent.com/assets/24825441/21718254/47706422-d43c-11e6-9eac-14c17520efca.png)

**Comment by waderyan** [comment]
Marketplace

**Comment by waderyan** [comment]
Please send to Marketplace team.
